src/components/Paginator/examples/5.text-field-props.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import createClass from 'create-react-class';
+import { Paginator } from '../../../index';
+
+export default createClass({
+	getInitialState() {
+		return {
+			selectedPageIndex: 0,
+		};
+	},
+	render() {
+		return (
+			<div>
+				<p>A paginator where textField is disabled using textFieldProps.</p>
+
+				<section>
+					<Paginator
+						hasPageSizeSelector
+						showTotalObjects
+						totalCount={12321313}
+						SingleSelect={{
+							DropMenu: { direction: 'up' },
+						}}
+						TextField={{
+							isDisabled: true,
+						}}
+					/>
+				</section>
+
+				<p>
+					A paginator where textField is disabled using textFieldProps and is
+					passed an index value.
+				</p>
+
+				<section>
+					<Paginator
+						hasPageSizeSelector
+						showTotalObjects
+						totalCount={12321313}
+						SingleSelect={{
+							DropMenu: { direction: 'up' },
+						}}
+						TextField={{
+							value: 3,
+							isDisabled: true,
+						}}
+					/>
+				</section>
+
+				<p>On submit</p>
+
+				<section>
+					<Paginator
+						hasPageSizeSelector
+						showTotalObjects
+						totalCount={12321313}
+						SingleSelect={{
+							DropMenu: { direction: 'up' },
+						}}
+						TextField={{
+							value: 3,
+							isDisabled: true,
```

Comment:
unless covered by another storybook example - also include an example if `isDisabled: false`  (assume it is covered as the default value)

Reply:
Covered by new storybook example